normits_demand/models/notem/production_models.py

```diff
@@ -306,19 +307,12 @@ def run(self,
             )
 
             if export_notem_segmentation:
-                du.print_w_toggle(
-                    "Exporting notem segmented demand to disk...",
-                    verbose=verbose
-                )
                 productions.to_pickle(self.export_paths.notem_segmented[year])
+                self._logger.info("Exporting notem segmented demand to disk...")
 
             if export_reports:
-                du.print_w_toggle(
-                    "Exporting notem segmented reports to disk...",
-                    verbose=verbose
-                )
-
                 notem_segmented_paths = self.report_paths.notem_segmented
+                self._logger.info("Exporting notem segmented reports to disk...")
```

Comment:
This seems to be a repeated pattern - please check this for all logs you've added in